src/browser-frontend/model/ui-page-model.js

```diff
@@ -14,6 +14,7 @@ import Immutable from 'immutable';
 
 const TAB_STATES = {
   INITIAL: 'initial',
+  ANIMATE: 'animate',
```

Comment:
Nice! Up to you, but I'd appreciate a more suggestive name for this. As it is, without knowing the rest of the code, I'm left asking the question "animate what?".